extensions/amp-analytics/0.1/vendors/snowplow_v2.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// To contribute, please submit issues and PRs to https://github.com/snowplow-incubator/amphtml
+
+import {jsonLiteral} from '../../../../src/json';
+
+const SNOWPLOW_V2_CONFIG = jsonLiteral({
+  'linkers': {
+    'linker': {
+      'ids': {
+        '_sp_duid': '${duid}',
+        'amp_id': '${ampVistorId}',
+      },
+    },
+  },
+  'cookies': {
+    'enabled': true,
+    'cookieMaxAge': 1800,
```

Comment:
To confirm, the cookeMaxAge unit is in second. It is the desired behavior to persist cookie for 30 mins?

Reply:
Ah no actually I want default value of 1 yr. Thanks for catching, commit on the way to fix.

Reply:
Got it, in that case, you don't need to specify the value. The default value is 365 days.

Reply:
Ah no actually I want default value of 1 yr. Thanks for catching!